use_cases/multi_tenant.rst

```diff
@@ -501,6 +501,11 @@ Next we move the data across the network to a new dedicated node. Create a new n
   -- move the shard to your choice of worker (it will also move the
   -- other shards created with the CASCADE option)
 
+  -- note that you should set wal_level for all nodes to be >= logical
```

Comment:
Should we include the steps to do this as part of our instructions? This comment has a lot packed inside.

Reply:
It's actually a requirement for `master_move_shard_placement` but I couldn't decide which one is better (comment or instruction), your call :)